src/Etu/Module/SIABundle/Form/createAccount.php

```diff
@@ -40,7 +40,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => true,
                 'mapped' => false,
                 'constraints' => new IsTrue(),
-                'label' => "J'accepte la transmission de mes informations personnelles nécéssaires à la création de mon compte (nom, prénom, adresse mail, numéro étudiant)",
+                'label' => "J'accepte la transmission de certaines de mes informations personnelles, nécessaires à la création de mon compte, au SIA. Ceci inclut: nom, prénom, adresse mail et numéro étudiant",
```

Comment:
```suggestion
                'label' => "J'accepte la transmission de certaines de mes informations personnelles, nécessaires à la création de mon compte, à l'UNG. Ceci inclut : nom, prénom, adresse mail et numéro étudiant",
```
C'est quand même ubuesque de se plaindre d'un manque de précision dans l'information délivrée, pour ensuite être encore moins précis. Le SIA n'a pas d'existence juridique, seule l'UNG en a une. Au demeurant, si on parlait bien du SIA, la destination était explicitée dans le reste de la page, ce qui rendait inutile cette modification.